### DISCUSSION TITLE
申请成为member

### DISCUSSION THREAD

**Discussion by leemos-xx** [mosn/layotto#675] [General]
1、自我介绍
工作大概十年左右的时间，之前做研发较多，接触的主要是分布式和微服务相关技术体系，近一年左右开始从事云计算行业。结合自己的经历，给自己后续制定的方向大概是朝云原生领域走，希望能从优秀的开源项目中学习的同时，也贡献一点微博的力量。
2、提交的pr
目前提交的PR都是与飞船计划相关的：
已完成：

- https://github.com/mosn/layotto/pull/658
- https://github.com/mosn/layotto/pull/665

进行中：

- https://github.com/mosn/layotto/issues/611

3、后续意向
先从简单的任务入手，等对mosn相关项目和体系熟悉后，希望能参与更重要的设计、开发任务。

**Comment by seeflood** [comment]
欢迎大佬加入社区，感谢贡献！！
![image](https://user-images.githubusercontent.com/26001097/176083948-98ca6b90-c0a9-402d-97f9-999883c08e28.png)
给你发邀请啦，可以看看邮箱

**Reply by seeflood** [reply]
可以进社区会议的钉钉群，群号：41585216
每周六晚8点有社区会议，感兴趣的话可以参与

**Comment by zhenjunMa** [comment]
👏👏👏
